### ISSUE TITLE
Integrate forms designer in backend .sdp package

### ISSUE THREAD

**Issue by mariuspopovici** [mariuspopovici/starlimsvscode#137]
- Merge form designer in the backend .sdp package
- Write post import script to deploy images and execute configuration tasks for designer pre-requisites
- Refactor the control properties install script to run on Oracle as well

**Comment by mariuspopovici** [comment]
Working on this

**Comment by mariuspopovici** [comment]
Committed changes in this branch https://github.com/mariuspopovici/starlimsvscode/tree/forms-desiger-01 

I integrated the forms designer in the backend SCM_API package. 

Added a post install script for the package which deploys the designer images. There's a limitation though here, the function that gets the webroot folder will return a different path if the .sdp is imported from the designer. This is because the designer import is executed in a SubmitToBatch() call which uses a different worker process executable.

This is not an issue when it's deployed from the API, during the extension installation.

Also refactored the control properties script so it works on Oracle as well. I put all the row values in an array and everything gets inserted in bulk with one RunSql call.

I'll do a PR with the changes at some point so this is merged in the next build.
